### PR TITLE
Add a interface-status annotation to pod

### PR DESCRIPTION
**Pull request by liuyuan10**
It holds information stored in NetworkInterface object. NetworkInterface will be replaced by this.

### CODE REVIEW

**Comment by liuyuan10** [comment]
@shouri007 Please review

File: apis/network/v1/annotations.go [hunk 1]
```diff
@@ -177,6 +183,33 @@ type NorthInterface struct {
 	IpAddress string `json:"ipAddress"`
 }
 
+// InterfaceStatus holds information of a NIC of a pod.
+// It's to replace NetworkInterface object.
```
Comment (shouri007) [inline apis/network/v1/annotations.go:187]: May be we don't need this statement in this description

**Review by shouri007 — APPROVED** [review]
LGTM

**Comment by liuyuan10** [comment]
@bowei could you approve this change given @mskrocki is not around?

**Pull request merged**